notify-action.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+NOTIFY_ARGS=(--session "path=/org/freedesktop/Notifications,member=ActionInvoked")
+
+# set -x
+
+dbus-monitor "${NOTIFY_ARGS[@]}" |
```

Comment:
Looks like `dbus-montitor` comes from `dbus` package while `gdbus` used in notify-send.sh is from `glib2` (`libglib2.0-bin` in Debian/Ubuntu). Let's mention `dbus-monitor` from `dbus` package dependency in README.md.

Reply:
I added a word about it on the README.md